### PR TITLE
fixing hover effect on cards on News page

### LINKED ISSUE #4470: [UI] hover effect on cards on news page
### Description

on https://layer5.io/company/news on hover on cards, the image gets cut and does not look appropriate.

### Expected Behavior

left and right margin can be added to the image so that even if it enlarges on hover, it would not get cut.

### Screenshots

image
image

### Environment:
- Host OS:
- Browser:

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by VamshiReddy02**
**Description**

This PR fixes #4470

I'm sorry for closing my Last PR because of some technical issues. As you all suggested I added Margin to the images and reduced their size also, please let me know of any changes.

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 34515fa159adebe19c960407cc2e4ced757fd6ae at: https://64ccac8c0ad7296318b84040--layer5.netlify.app

**Comment by Ghat0tkach** [comment]
Hey @VamshiReddy02 

 Let's discuss it on the websites call.
Please add this as an agenda item in the meeting minutes, if you would :)

https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima

**Review by hirentimbadiya — APPROVED** [review]
LGTM 👍🏻

**Review by abhijeetgauravm — APPROVED** [review]
LGTM !

**Review by HetviSoni — CHANGES_REQUESTED** [review]
+1 to @abhijeetgauravm 's point and also one more. The bottom of this image is getting cut even before we hover 
image

**Comment by l5io** [comment]
🚀 Preview for commit ecbd9f7b7d06e0f0afb0df196ea766eedc275b2b at: https://64d516ba8eac69031382dbd6--layer5.netlify.app

**Comment by VamshiReddy02** [comment]
Hey @abhijeetgauravm and @HetviSoni, I made some changes. Please Review my changes again and please let me know of any changes.

File: src/components/Card/Card.style.js [hunk 1]
```diff
@@ -41,16 +41,23 @@ export const CardWrapper = styled.div`
         border-top-right-radius: 0.5rem;
         border-top-left-radius: 0.5rem;
 
-        .gatsby-image-wrapper, .old-gatsby-image-wrapper{
+        .gatsby-image-wrapper{
+            height:80%;
+            width:70%;
+            margin-top: 0.5rem;
+            margin-left: 3rem;
```
Comment (HetviSoni) [inline src/components/Card/Card.style.js:48]: Here you have explicitly added margin-left as 3rem which is causing alignment issues on some screen size between desktop and mobile(image is not in center of the card).

Maybe you can try 
`margin:auto;
margin-top:0.5rem; `
image

Comment (HetviSoni) [reply src/components/Card/Card.style.js:48]: Well good work @VamshiReddy02 it is working for the desktop screen.

**Comment by l5io** [comment]
🚀 Preview for commit 2bd692f94264bdcfd0edaee68582c2c07224de38 at: https://64d529b3903428009dc8d570--layer5.netlify.app

**Review by HetviSoni — APPROVED** [review]
Good work @VamshiReddy02 😄

**Review by abhijeetgauravm — APPROVED** [review]
Looks Good @VamshiReddy02

**Review by thisiskaransgit — CHANGES_REQUESTED** [review]
@VamshiReddy02 Not quite,
- too much padding in directions where it was not needed
- updating classes' styles that are assigned to multiple elements will have effects, go to /resources, and /blog sections to review the changes.

We needed to specifically update the particular section's elements. Since the images are of different dimensions, we cannot ensure to prevent image overflow for each of them.  

@VamshiReddy02 @HetviSoni, please conclude this task by this week

**Comment by l5io** [comment]
🚀 Preview for commit 99ddeb81b1618839c4ae10ae3c9133ead7d38775 at: https://64d68fbcfaf31907e05eeff1--layer5.netlify.app

**Comment by HetviSoni** [comment]
LGTM now

**Comment by vishalvivekm** [comment]
@VamshiReddy02 Let's discuss it on the websites call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima), if you would :)

**Comment by VamshiReddy02** [comment]
hello everyone, please let me know of any changes.

**Review by HetviSoni — APPROVED** [review]

**Review by Chadha93 — APPROVED** [review]
Thanks @VamshiReddy02

**Pull request merged**